stream/src/public_key/benches/sender_send.rs

```
@@ -22,11 +22,11 @@ fn benchmark_sender_send(c: &mut Criterion) {
                         cipher1,
                         cipher2,
                     );
-                    let (sender, _receiver) = connection.split();
```

Dropping the receiver here caused the benchmark to silently fail.